OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -1243,75 +1240,76 @@ namespace OloEngine
         return *s_DefaultMaterial;
     }
 
-    void Scene::RenderScene3D(EditorCamera const& camera)
+    void Scene::LoadAndRenderSkybox()
```

Clang (14.0.6)

[warning] method 'LoadAndRenderSkybox' can be made static

(readability-convert-member-functions-to-static)